src/features/commands/command-line-hook.tsx

```diff
@@ -117,12 +119,22 @@ export const CommandLineComponent: React.FC<{
   const [selectedActionIndex, setSelectedActionIndex] = useState(0)
   const [firstActionIndex, setFirstActionIndex] = useState(0)
 
+  useAdvancedActions(
+    searchValue,
+    (value: string) => setSearchValue(value),
+    (value: CommandLineAction[]) => setSearchResult(value),
```

Comment:
Don't have to specify types here, they are already defined internally (and your IDE will hint them anyway)

```suggestion
    (value) => setSearchValue(value),
    (value) => setSearchResult(value),
```

Reply:
And also, what do you think about my previous comment on making the input an interface instead of 4 parameters?